packages/react/src/themes/teams/components/Dropdown/dropdownStyles.ts

```diff
@@ -3,23 +3,49 @@ import { DropdownProps, DropdownState } from '../../../../components/Dropdown/Dr
 import { DropdownVariables } from './dropdownVariables'
 import { pxToRem } from '../../../../lib'
 
-const dropdownStyles: ComponentSlotStylesInput<DropdownProps & DropdownState, DropdownVariables> = {
-  root: (): ICSSInJSStyle => ({}),
+type Props = DropdownProps & DropdownState
+
+const transparentColorStyle: ICSSInJSStyle = {
+  backgroundColor: 'transparent',
+  borderColor: 'transparent',
+  borderBottomColor: 'transparent',
+}
+
+const transparentColorStyleObj: ICSSInJSStyle = {
+  ...transparentColorStyle,
+  ':hover': transparentColorStyle,
+  ':active': transparentColorStyle,
+  ':focus': {
+    ...transparentColorStyle,
+    ':active': transparentColorStyle,
+  },
+}
+
+const getWidth = (p: Props, v: DropdownVariables): string =>
+  p.fluid ? '100%' : p.inline ? 'initial' : v.width
```

Comment:
https://eslint.org/docs/rules/no-nested-ternary

Reply:
done, but we should have this as part of the build process or in dedicated `eslint` so that we get a warning while using it
@layershifter

Reply:
#600, will check it, actually should be unblocked now :)